app/gui2/src/components/SelectionBrush.vue

```diff
@@ -40,7 +40,11 @@ const brushStyle = computed(() => {
 </script>
 
 <template>
-  <div class="SelectionBrush" :class="{ hidden }" :style="brushStyle"></div>
+  <div
+    class="SelectionBrush"
+    :class="{ cursor: props.anchor == null, hidden }"
```

Comment:
Why do we need class for cursor? It looks like bringing back our own cursor instead of using system one.